grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -0,0 +1,2441 @@
+apiVersion: v1
+data:
+  SLOs.json: |-
```

Comment:
This is configmap yaml, but now this file should be removed from PR

Reply:
Ok, I was wondering why there was 2. I don't know how I committed that to root in the first place, rushing on a Friday! Back to one file now. Thanks.